From 77e2c3fee87eab24316d9722400183cb192a653a Mon Sep 17 00:00:00 2001
From: yangz1x <zhonghuax.yang@intel.com>
Date: Wed, 11 Sep 2019 09:53:09 +0800
Subject: [PATCH 1500/1552] media: intel-ipu4: css: scci_IPU4_master_20190813
 fw release

FW tag : scci_stable_iot_mr5_ivi_20190726_20190819_1700

Change-Id: Icaffd40ec3f000bdce5a6b54074d2506adaa55bd
Signed-off-by: icg_intg <icg_intg@intel.com>
Tracked-On: PKT-2639
---
 drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h          | 2 +-
 .../pci/intel/ipu4/ipu4-css/lib2600/config/isys/subsystem_bxtB0.mk     | 3 +++
 .../lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h       | 2 +-
 3 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
index b939cf7..9a60f1a 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20190320
+#define IA_CSS_FW_PKG_RELEASE  0x20190813
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/config/isys/subsystem_bxtB0.mk b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/config/isys/subsystem_bxtB0.mk
index da14203..c4549f3 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/config/isys/subsystem_bxtB0.mk
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/config/isys/subsystem_bxtB0.mk
@@ -46,6 +46,9 @@ WA_HSD1209062354		 = 1
 # WA to disable clock gating for the devices in the CSI receivers needed for using the mipi_pkt_gen device
 WA_HSD1805168877		 = 0
 
+# WA to match the NofSOF & NofEOF when resuming a stream to prevent fps drop
+WA_HSD1507026600         = 1
+
 # Support IBUF soft-reset at stream start
 SOFT_RESET_IBUF_STREAM_START_SUPPORT = 1
 
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
index b939cf7..9a60f1a 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20190320
+#define IA_CSS_FW_PKG_RELEASE  0x20190813
-- 
2.7.4

